### ISSUE TITLE
A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard:Keyboard focus is not going to the circular list items present in the carousel under heading "Visual Studio Code in Action"

### ISSUE THREAD

**Issue by yoginigorle** [microsoft/vscode-docs#5541]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win11;#FTP;#DesktopWeb;#A11ySev1;#WCAG2.1.1;#Keyboard;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win11 Version 21H2(OS Build 22000.795)
Chrome Version: Version 103.0.5060.134 (Official Build) (64-bit)

## Repro Steps:

1. **Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix**
2. Launch the application Url : https://code.visualstudio.com/
3. Navigate to "Download" button.
4. Hit on any of the platform(Windows/Linux/MacOS) links or any installer type(*64 and *32).
5. Thanks for downloading VS code for windows screen will open.
6. Tab to the list items under heading "Visual studio code in action".

## Actual:
Keyboard focus is not going to the circular list items present in the carousel under heading "Visual Studio Code in Action"

## Expected:
Keyboard focus should go to each of the list items under heading "Visual studio code in action".First list item should be accessible with the tab key and the rest should be accessible with arrow keys.

## User Impact:
Keyboard users will not be able to access the list items.

## Attachment
![5541_A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard](https://user-images.githubusercontent.com/105736242/183053732-7fbb3fcc-0b72-4bb9-9a2a-909af33f7a23.gif)

**Comment by gregvanl** [comment]
@yoginigorle The carousel control works as expected for me. I can tab into the "Visual Studio Code in Action" carousel and use the arrow keys to shuffle the slides back and forth.
Closing as GitHub issues do not have a "Resolved" state.

**Comment by kapilvaishna** [comment]
GithubTags: #Closed;
 Not repro in environment https://code.visualstudio.com/
 

https://user-images.githubusercontent.com/124254981/216300088-36838387-40b7-4de6-b3cd-e22e1a65cb29.mp4
